src/app/Fake.IO.FileSystem/Operators.fs

```diff
@@ -0,0 +1,9 @@
+namespace Fake.IO.FileSystem
+
+open System.IO
+
+module Operators =
```

Comment:
If we move to `Fake.IO` I'm not sure how we should call this. `Fake.IO.Operators` might be too generic. Maybe we leave it in this namespace or call it `PathOperators`?

Reply:
I will change namespace to File.IO. And for Operators, PathOperators seems good ? or FileSystemOperators ? (since we are in Fake.IO.FileSystem package)